certified-connectors/SmartCOMM On Demand Doc-Gen/Readme.md

```diff
@@ -0,0 +1,48 @@
+## SmartCOMM On Demand Generation Connector
```

Comment:
Please review your readme file and, format and add the required sections. For proper formatting, you can always refer to this [template](https://github.com/microsoft/PowerPlatformConnectors/tree/dev/templates/certified-connectors).

Reply:
The readme.md file's format and require sections have been updated (following the reference template structure) and is now ready for review.